companies/tasks/__init__.py

```
@@ -1 +1,7 @@
-# companies/tasks/__init__.py
+# Companies tasks
+# 태스크를 명시적으로 import하여 Celery가 자동으로 발견할 수 있도록 함
+from . import financial_metrics  # noqa: F401
```

Ruff (0.14.13)

Unused `noqa` directive (non-enabled: `F401`)

Remove unused `noqa` directive

(RUF100)